src/main/java/org/grouplens/grapht/Instantiators.java

```diff
@@ -209,10 +226,12 @@ public InstantiatorProvider(Instantiator itor) {
         @Override
         public Object get() {
             try {
+                Logger.debug("Casting Instantiator ",instantiator.instantiate());
```

Comment:
I don't think we need to log that we are casting; if we do include a log message here, it should be that we are invoking the instantiator, and it should be a `trace`-level message.

Also, we should log the instantiator, not the result of instantiating it (that should only happen once).

And we need to include '{}' as a placeholder in the string to include the object in the log message.